src/test/testng-travis.xml

```diff
@@ -28,68 +29,68 @@
+            <class name="teammates.test.cases.browsertests.AdminEmailLogPageUiTest" />
+            <class name="teammates.test.cases.browsertests.AutomatedSessionRemindersTest" />
+            <class name="teammates.test.cases.browsertests.FeedbackConstSumOptionQuestionUiTest" />
+            <class name="teammates.test.cases.browsertests.FeedbackConstSumRecipientQuestionUiTest" />
+            <class name="teammates.test.cases.browsertests.FeedbackContributionQuestionUiTest" />
+            <class name="teammates.test.cases.browsertests.FeedbackMcqQuestionUiTest" />
+            <class name="teammates.test.cases.browsertests.FeedbackMsqQuestionUiTest" />
+            <class name="teammates.test.cases.browsertests.FeedbackNumScaleQuestionUiTest" />
+            <class name="teammates.test.cases.browsertests.FeedbackRubricQuestionUiTest" />
+            <class name="teammates.test.cases.browsertests.FeedbackRankQuestionUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorEditStudentFeedbackPageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorFeedbackEditCopyUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorFeedbackEditPageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorFeedbackPageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorFeedbackResultsPageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorFeedbackSubmitPageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorStudentListPageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorStudentRecordsPageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorSubmissionAdjustmentUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorHomePageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorSearchPageUiTest" />
+            <class name="teammates.test.cases.browsertests.StudentCourseDetailsPageUiTest" />
+            <class name="teammates.test.cases.browsertests.StudentFeedbackSubmitPageUiTest" />
+            <class name="teammates.test.cases.browsertests.StudentFeedbackResultsPageUiTest" />
+            <class name="teammates.test.cases.browsertests.StudentProfilePageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorCourseDetailsPageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorCommentsPageUiTest" />
+            <class name="teammates.test.cases.browsertests.StudentCommentsPageUiTest" />
+            <class name="teammates.test.cases.browsertests.AdminEmailPageUiTest" />
+            <class name="teammates.test.cases.browsertests.AdminSessionsPageUiTest" />
+            <class name="teammates.test.cases.browsertests.AdminSearchPageUiTest" />
```

Comment:
Unrelated to this PR. A better option is to put the test class in the same package as the SUT so that the test class can access package level members of the SUT. However, tests classes should reside in a separate physical location from production code.